### PR TITLE
Remove Gitlab linkage

### PR DESCRIPTION
**Pull request by bocekm**
- Purpose of linking GitHub with Gitlab was to run unit tests on Fedora which is no longer relevant. _tox.ini_ was part of the linkage.
- Remove obsolete _requirements.txt_ - _six_ can be installed using _pip_.
- Remove _coverage_ related code/files - it hasn't been run for a long time and I don't think we need that.
- Add info to _README.md_ about how to build the package and run tests.

### CODE REVIEW

**Comment by bocekm** [comment]
--test--

**Comment by bocekm** [comment]
--test--

File: README.md [hunk 1]
```diff
@@ -1,45 +1,37 @@
...
 
-The Preupgrade Assistant performs an assessment of the system from the "upgradeability" point of view.
+The Preupgrade Assistant is a diagnostics utility that assesses the system for possible in-place upgrade limitations and provides a report with the analysis results. It is based on a module system, with each module performing a separate test, checking for package removals, incompatible obsoletes, changes in libraries, name changes, or deficiencies in the compatibilities of certain configuration files. The data gathered by the Preupgrade Assistant can be used for cloning the system. It also provides post-upgrade scripts to finish more complex problems after the in-place upgrade. The Preupgrade Assistant utility is a prerequisite for completing a successful in-place upgrade to the next major version of Red Hat Enterprise Linux. The data gathered by the Preupgrade Assistant is required by [Red Hat Upgrade Tool](https://github.com/upgrades-migrations/redhat-upgrade-tool), which performs the in-place upgrade of the system.
```
Comment (pirat89) [inline README.md]: The description still contains information that is more relevant to modules. More then "*it also provides post-upgrade scripts*" would be better something like "*it also provides posibility to process **post-upgrade scripts** that can be provided by modules*"

Comment (bocekm) [inline README.md]: I agree, I've created a new issue to fix the description: https://github.com/upgrades-migrations/preupgrade-assistant/issues/295

**Pull request merged**